stock_picking_invoice_link/views/stock_view.xml

```diff
@@ -8,11 +8,13 @@
             <field name="model">stock.picking</field>
             <field name="inherit_id" ref="stock.view_picking_form"/>
             <field name="arch" type="xml">
-                <notebook position="inside">
-                    <page string="Invoices" groups="account.group_account_invoice">
-                        <field name="invoice_ids" nolabel="1" context="{'tree_view_ref': 'account.invoice_tree', 'form_view_ref': 'invoice_form'}"/>
-                    </page>
-                </notebook>
+                <field name="picking_type_id" position="after">
+                    <field name="invoice_ids" string="Invoice"
+                        groups="account.group_account_invoice"
+                        attrs="{'invisible':[('invoice_ids', '=', False)]}"
+                        widget="many2many_tags"
+                        context="{'tree_view_ref': 'account.invoice_tree', 'form_view_ref': 'invoice_form'}"/>
```

Comment:
As explain in this pr https://github.com/OCA/stock-logistics-workflow/pull/319/files , full reference is required so customer invoice form is correctly displayed. Use "stock_picking_invoice_link.invoice_form" instead of "invoice_form".